cache/src/main/java/com/uber/buckcache/resources/buckcache/BuckCacheResource.java

```diff
@@ -89,7 +82,7 @@ public String getSummary() throws Exception {
   @Produces(MediaType.APPLICATION_OCTET_STREAM)
   public Response getCacheArtifact(@PathParam("key") String key,
                                    @HeaderParam(X_CACHE_TAGS) String cacheTags) throws Exception {
-    StatsDClient.get().count(getCacheKeyWithCustomizedParam(GET_CALL_COUNT, cacheTags), 1L);
+    StatsDClient.get().count(GET_CALL_COUNT, 1L, cacheTags);
```

Comment:
There is some code duplication with `cacheTags` being passed for each reported metric.
We can redesign `StatsDClient` to support setting cacheTags for a given scope and avoid the duplication.
But it doesn't seem like worth the effort right now.

Let me know what your thoughts are.